bin/install

```diff
@@ -118,20 +162,34 @@ download_luaJIT_source() {
   local install_type=$1
   local version=$2
   local download_path=$3
-  local download_url=$(get_luaJIT_download_url)
+  local download_url
+  download_url=$(get_luaJIT_download_url)
 
-  echo clone luajit from ${download_url} into ${download_path}
-
-  git clone --branch=v$version $download_url $download_path
+  if ! [[ -d "$download_path" ]]; then
+    echo clone luajit from "${download_url}" into "${download_path}"
+    git clone --branch=v"$version" "$download_url" "$download_path"
+  else
+    if [[ $(git -C "$download_path" remote get-url origin) != $(get_luaJIT_download_url) ]]; then
```

Comment:
If we created this directory earlier, this should certainly be the case.

If not, something really funky happened, and we'd rather not go touch/moving/deleting files we weren't expecting to, so we give up.